modules/tinymce/src/core/test/ts/browser/selection/GetSelectionContentTest.ts

```diff
@@ -45,15 +48,15 @@ UnitTest.asynctest('browser.tinymce.selection.GetSelectionContentTest', (success
       }
     };
 
-    return GeneralSteps.sequence([
-      Step.sync(function () {
+    return Log.stepsAsStep('--', label, [
```

Comment:
We should use `Step.label` here instead of `Log.stepsAsStep`, as `Log` is for top-level entries.

Reply:
Yeah, I assumed there was something, but my attempt for `Step.stepsAsSteps` failed. This will help.

Reply:
When I actually tried it it seems as if it only takes a single step. Do we have anything but `GeneralSteps.sequence` and `Log.stepsAsStep` that makes the `step[]` -> `step` conversion?

Reply:
Yeah sorry, forgot to mention you'd need to use `GeneralSteps`, so what you've done atm is the best we have that's not deprecated. We probably should expand the API to provide a better replacement for `Logger.ts`.